tests/unit/commands/local/lib/test_api_provider.py

```diff
@@ -0,0 +1,174 @@
+from collections import OrderedDict
+from unittest import TestCase
+
+from mock import patch
+
+from samcli.commands.local.lib.api_provider import ApiProvider, SamApiProvider, CfnApiProvider
```

Comment:
Why this instead of importing them from the classes that the classes are defined in? Looks strange to me.